gatsby/assets/scss/design-system/layout/_navigation.scss

```diff
@@ -43,7 +43,7 @@
 				margin-right: auto;
 
 				// Media queries
-				@include media-breakpoint-up(sm) {
+				@include media-breakpoint-down(sm) {
```

Comment:
sice jsme se domlouvali, že nebudeme sahat do souborů design systému, ale asi je to jedno 😄 🤷‍♂️

Reply:
ah, vidis to :D ja jsem si tooho asi ani nevsiml 🤦‍♂️ 😄 no, co uz..